src/resolvers/git.cr

```diff
@@ -357,6 +362,10 @@ module Shards
 
     # Parses a URI string, with additional support for ssh+git URI schemes.
     private def parse_uri(raw_uri)
+      if (path = raw_uri.lchop?("file://"))
```

Comment:
Would be nice to have a comment on this as well.
Also what about `file:\\\\` URIs?

Reply:
Comment done.

I don't know, what about them? Backslashes are not valid here, and if there are 4 slashes, then I guess 2 will remain, and maybe Windows will still do something useful with them?
https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes.3F

Reply:
Sry, I wrote the backslashes escaped. I just meant `file:\\` followed by an anchored path. This is at least somewhat common on windows, although not valid as a URI.
Still, maybe adding `|| (path = raw_uri.lchop?("file:\\\\"))` wouldn't be a bad idea?